static/components/PoiCard.js

```diff
@@ -23,23 +24,32 @@ class PoiCard extends React.Component {
     })
     if(this.props.details._id === getHash()) {
       highlight(this.props.details._id)
-      masterCard(this.props.details._id)
+      superCard(this.props.details._id)
       firstCard(this.props.details._id)
     }
   }
 
-  calculateDistance(from, to) {
+  calculateDistance(from, to, typeDistance) {
```

Comment:
it's for 3 cases right now:
- `meters`
- `km`
or
empty in the `style='order: calculateDistance'` 

And also in the future (maybe with miles or others)